src/utils/notify.rs

```diff
@@ -31,11 +31,12 @@ pub async fn notify(bot: &StarboardBot, user_id: Id<UserMarker>, message: &str)
         })],
     });
 
-    let _ = create
+    create
         .content(message)
         .unwrap()
         .components(&[comp])
         .unwrap()
         .exec()
-        .await;
+        .await
+        .expect("");
```

Comment:
Won't this cause a panic? I would prefer it not to panic on failure.

Reply:
idk im not an expert

Reply:
Yeah I tested, this will cause a panic instead of failing silently. Could you revert this change

Reply:
Sure :+1: